readme.txt

```diff
@@ -223,6 +223,9 @@ Remember to always make a backup of your database and files before updating!
 
 = [5.6.1] TBD =
 
+<<<<<<< HEAD
```

Comment:
Fix this merge conflict and this is coming along nicely!

Reply:
🤦🏼